### DISCUSSION TITLE
How to restart killed flink cdc job without GTIDs & Snapshot?

### DISCUSSION THREAD

**Discussion by SML0127** [apache/flink-cdc#2174] [Q&A]
Hello, I'm a newbie of flink mysql-cdc :D

I have a question about restarting flink cdc job. 🙇🙇‍♀️🙇🏻‍♂️

Suppose the cdc job is killed after finishing snapshot.
Then, I just want to capture all event after killed without snapshot.
I know that using `specific-offset` mode with GTIDs (latest GTID before killed), it can capture all changes after killed.

But, is there any `startup.mode` or function that allows to capture changes only with flink checkpoint and without GTID?
It is burdensome to check the GTIDs in DB or Timestamp every time...🥲

**Comment by mamineturki** [comment]
any updates on this problem please?
